### ISSUE TITLE
Where is the decay strategy of stride in the code ?

### ISSUE THREAD

**Issue by chobao** [VITA-Group/SinNeRF#12]
Hi, Thanks for releasing the code of the awesome work. I am curious about the decay strategy of stride in _Progressive Strided Ray Sampling_. However, I can not find the correct place in the code. It seems that the stride is controlled by `sW,sH` in `Dataset` class but `sW,sH` are not modified during training. The same problem happens for the `vit_weight` and `dis_weight`, which are weights of `global structure prior ` loss and  ` local texture guidance` loss.

The fragment in the paper about the`decay strategy of stride`:
<img width="782" alt="image" src="https://user-images.githubusercontent.com/53040496/180960764-1632ed4e-da12-45fb-9b44-9473f2683e4a.png">

The fragment in the paper about the `annealling of loss weights`:
<img width="772" alt="image" src="https://user-images.githubusercontent.com/53040496/180961204-7593f00d-97fc-4bd4-87fc-2514780ba2dc.png">

**Comment by Ir1d** [comment]
Hi, we specify different hyperparameters during the training of multiple stages. For example, for the first stage of lego, we use stride=6, and for later stages, we use stride=4, 2, 1.

**Comment by chobao** [comment]
Oh, Thanks. I forgot to read README. Therefore, the stride will not decrease at each stage.
